SPECS/openssh/openssh.spec

```diff
@@ -93,6 +94,7 @@ The module is most useful for su and sudo service stacks.
 %prep
 %setup -q -a 3
 
+%patch308 -p2 -b .cve-2023-38408
```

Comment:
Can you move the patch so it follows sequentially?  or does it need to be here

Reply:
updated.